shell/src/main/java/alluxio/cli/fs/command/DuCommand.java

```diff
@@ -38,13 +42,16 @@
 @ThreadSafe
 @PublicApi
 public final class DuCommand extends AbstractFileSystemCommand {
-  private static final String LONG_INFO_FORMAT = "%-13s %-16s %-16s %s";
+  private static final String LONG_INFO_FORMAT = "%-13s %-16s %-16s %-25s %s";
+  private static final String MID_1_INFO_FORMAT = "%-13s %-16s %-16s %s";
+  private static final String MID_2_INFO_FORMAT = "%-13s %-16s %-25s %s";
```

Comment:
What is the difference between MID_1 and MID_2? I think maybe you can add comments to each with an example, like below, and the "aaa bbb" are one sample output.
```
  // aaaaaaaa bbbb cccc dddd e
  private static final String LONG_INFO_FORMAT = "%-13s %-16s %-16s %-25s %s";
  // aaaaaaaa bbbb cccc dddd
  private static final String MID_1_INFO_FORMAT = "%-13s %-16s %-16s %s";
  // aaaaaaaa bbbb cccc dddddddddd
  private static final String MID_2_INFO_FORMAT = "%-13s %-16s %-25s %s";
```